src/js/components/HealthTab.js

```diff
@@ -132,7 +132,7 @@ class HealthTab extends React.Component {
         <FilterHeadline
           currentLength={visibleData.length}
           inverseStyle={true}
-          name={"Health Checks"}
+          name={"Health Check"}
```

Comment:
Should we pluralize this based off of `units.getItems().length` (the total number of health checks)?